src/plugins/preauth/pkinit/pkinit_crypto_openssl.c

```diff
@@ -39,15 +39,16 @@
 #include <arpa/inet.h>
 
 #if OPENSSL_VERSION_NUMBER >= 0x30000000L
-#include <openssl/core_names.h>
+#include <openssl/evp.h>
```

Comment:
Why this change?

(There is some accumulated debt here because two of these includes are also done in pkinit_crypto_openssl.h.  And that header should probably go away because it's only included by this one source file.  But I don't think either of those problems is related to this PR.)

Reply:
I remove the inclusion of `openssl/evp.h`, it was already imported in `pkinit_crypto_openssl.c` indeed. It is the same for `openssl/core_names.h`, so I removed it too.

Reply:
Maybe you prefer include operations to remain untouched in this PR. It that's the case, just let me know and I will leave it as it was.

Reply:
Let's leave them alone for now (assuming I'm correct that they aren't needed for the PR), and I'll plan to do a cleanup PR (folding the header file into the source file) after this and the ECC work are done.